ArmVirtPkg/ArmVirtQemu.dsc

```diff
@@ -102,16 +102,16 @@
 
 [LibraryClasses.common.PEIM]
   ArmVirtMemInfoLib|ArmVirtPkg/Library/QemuVirtMemInfoLib/QemuVirtMemInfoPeiLib.inf
-
-!if $(TPM2_ENABLE) == TRUE

```

Comment:
NIT: Should the `!if $(TPM2_ENABLE) == TRUE` be kept ?

Reply:
I'm not sure why I added those to begin with: there is nothing wrong with providing a library class resolution that remains unused if TPM2_ENABLE happens to be unset.